### ISSUE TITLE
Implement "show_FullHeader" control over header width

### ISSUE THREAD

**Issue by grumpyinca** [thegrumpys/odop#327]
Transferred from #219: 

Experiments in operating ODOP version 3.2.3 on my 720x1280 pixel cellphone lead me to believe that there is a need for an option to limit the header width.  

I propose show_FullHeader = 1 is the default and displays what is in place today.  show_FullHeader = 0 could display a width-limited subset.  Specifically, It would eliminate display of the ODOP logo and abbreviate tab names.  The "Design:" tab name could become "D:".  The "Report: 1 (mini)" tab name could become "R1", etc.

This change should reduce the occurrence of header line-wrap reducing the vertical screen space available for the rest of the page.  Vertical screen space available to the body of the main page becomes significant (critical!) when using the phone in landscape mode and also in entry mode (the virtual keyboard is displayed).

**Comment by grumpyinca** [comment]
Just f.y.i. ... The logic inside the reports module is dependent on the report name.  I can certainly handle that part if it helps.

**Comment by 1fifoto** [comment]
Today we prefix the design with "Design: " followed by the design type image followed by the design name. What do you think about just displaying only the design name on small screens.

Similarly we prefix the reports with "Report: " followed by the report name. What do you think about just displaying only the report name on small screens.

For example...
![image](https://user-images.githubusercontent.com/1581602/73888158-b7f3e300-4832-11ea-9737-a56e56b76597.png)

In other words I'm not feelin' "show_FullHeader = 1". Can you try this out on your phone and tell me if it does what you want. It might not because your screen is 720x1280 and that might be considered medium or large.

**Comment by grumpyinca** [comment]
I can't try it on my phone until it gets pushed as far as Heroku.  I don't see any reason why the current implementation can't get pushed out in the 3.3 release.  If it doesn't get the job done, we can always try again in the following release (3.3.1).

Are the reports still working ?

**Comment by 1fifoto** [comment]
Merged and closing
